rollup-cli/tools/resourcesBot/walletFunder.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
this file should be in `.gitignore`
We could add some examples files but it should be pointed out on file name: `walletFunder.example.json`
:smile: